Lib/test/test_subprocess.py

```diff
@@ -2195,7 +2193,7 @@ def _check_swap_std_fds_with_one_closed(self, from_fds, to_fds):
                         os.write(fd, str(fd).encode('ascii'))
             ''')
 
-            skipped_fd = next(fd for fd in range(3) if fd not in to_fds)
+            skipped_fd = (set(range(3)) - set(to_fds)).pop()
```

Comment:
I've used the suggested construct but with `range(3)` instead of `{0, 1, 2}` for consistency with the surrounding code.